<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpinBox</class>
 <widget class="QWidget" name="SpinBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;</string>
  </property>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	background-color: #FFFFFF;
	border-style: solid;
	border-color: #000000;
	border-width: 1px;
	width: 38 px;
	height: 38 px;
	subcontrol-origin: border;
	subcontrol-position: top right;
}

QPushButton:pressed{

	background-color: #ECEFF1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/Icons/UI/Icons/sharp-add-24px.svg</normaloff>:/Icons/UI/Icons/sharp-add-24px.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueLine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{

	border-color: #000000;
	border-radius: 0px;
	border-width: 1px;
	border-style: solid;
	color: #000000
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>41</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	background-color: #FFFFFF;
	border-style: solid;
	border-color: #000000;
	border-width: 1px;
	width: 38 px;
	height: 38 px;
	subcontrol-origin: border;
	subcontrol-position: top right;
}

QPushButton:pressed{

	background-color: #ECEFF1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/Icons/UI/Icons/sharp-remove-24px.svg</normaloff>:/Icons/UI/Icons/sharp-remove-24px.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
